sdstore/cache2disk.go

```diff
@@ -293,16 +299,16 @@ func Cache2Disk(command, cacheScope, srcDir string, compress, md5Check bool, cac
 	if strings.HasPrefix(baseCacheDir, "~/") {
 		baseCacheDir = filepath.Join(homeDir, strings.TrimPrefix(baseCacheDir, "~/"))
 	}
-
 	if strings.HasPrefix(srcDir, "~/") {
 		srcDir = filepath.Join(homeDir, strings.TrimPrefix(srcDir, "~/"))
 	}
-
-	if srcDir, err = filepath.Abs(srcDir); err != nil {
-		msg = fmt.Sprintf("%v in src path %v, command: %v", err, srcDir, command)
-		return logger.Log(logger.LOGLEVEL_ERROR, "", logger.ERRTYPE_FILE, msg)
+	srcDir = filepath.Clean(srcDir)
+	if strings.HasPrefix(srcDir, "../") {
```

Comment:
what is the cwd this `../` relative to?

Reply:
don't know, that's why getting an absolute path.

Reply:
> If the path is not absolute it will be joined with the current working directory to turn it into an absolute path

for cache, is the srcDir restricted to be the actual source dir or it could be anywhere from `/`?

Reply:
actual source dir